packages/manager/modules/config/src/environment/environment.service.js

```diff
@@ -29,12 +31,17 @@ export default class EnvironmentService {
 
   setUser(user) {
     this.user = user;
+    this.supportLevel = new SupportLevel(user.supportLevel);
```

Comment:
```suggestion
  setUser(user) {
    this.user = {
    ...user, 
    supportLevel: new SupportLevel(user.supportLevel);
    }
```
Not sure supportLevel should be a direct property of `Environment`

Reply:
Right, i removed the property.
Fixed.